tools/run_tests/xds_k8s_test_driver/README.md

```diff
@@ -269,7 +227,44 @@ envsubst < config/local-dev.cfg.example > config/local-dev.cfg
 
 Learn more about flagfiles in [abseil documentation](https://abseil.io/docs/python/guides/flags#a-note-about---flagfile).
 
-### Helper scripts
+## Test suites
+
+See the full list of available test suites in the [`tests/`](https://github.com/grpc/grpc/tree/master/tools/run_tests/xds_k8s_test_driver/tests) folder. 
+
+### xDS Baseline Tests
+
+Test suite meant to confirm that basic xDS features work as expected. Executing
+it before other test suites will help to identify whether test failure related
+to specific features under test, or caused by unrelated infrastructure
+disturbances.
+
+```shell
+# Help
+python -m tests.baseline_test --help
+python -m tests.baseline_test --helpfull
+
+# Run the baseline test with local-dev.cfg settings
+python -m tests.baseline_test --flagfile="config/local-dev.cfg"
+  
+# Same as above, but using the helper script
+./run.sh tests/baseline_test.py
```

Comment:
optional: I would recommend to layout all the command line arguments, so the people onboarding the framework can see what's getting configured here. The `./run.sh` can exist as a shortcut documented as an alternative way of running the tests.

Reply:
People onboarding are likely to use `./run.sh` - that's from my experience of a few folks recently. Regarding command arguments, there are a lot of them. That's why I'm suggesting `--help` above.

But I think we could list all *required* arguments. Would it be ok if this is tackled outside of this PR?